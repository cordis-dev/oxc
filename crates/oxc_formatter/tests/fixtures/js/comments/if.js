if (base.endsWith('.js') || base === `/worker-entries`); // for dev
if (base.endsWith('.js') || base === `/worker-entries`) base = '' // for dev
if (base.endsWith('.js') || base === `/worker-entries`) a; // for dev
