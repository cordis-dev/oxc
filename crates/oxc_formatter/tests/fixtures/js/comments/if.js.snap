---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
if (base.endsWith('.js') || base === `/worker-entries`); // for dev
if (base.endsWith('.js') || base === `/worker-entries`) base = '' // for dev
if (base.endsWith('.js') || base === `/worker-entries`) a; // for dev

==================== Output ====================
if (base.endsWith(".js") || base === `/worker-entries`); // for dev
if (base.endsWith(".js") || base === `/worker-entries`) base = ""; // for dev
if (base.endsWith(".js") || base === `/worker-entries`) a; // for dev

===================== End =====================
